internal/services/consumption/consumption_budget_management_group_resource.go

```diff
@@ -77,6 +77,13 @@ func (r ManagementGroupConsumptionBudget) Arguments() map[string]*pluginsdk.Sche
 						}, false),
 					},
 
+					"locale": {
+						Type:         pluginsdk.TypeString,
+						Optional:     true,
+						Computed:     true, // TODO: make this required in 4.0
```

Comment:
we will not be accepting any more todo next major version comments like this post 4.0. We should have/be adding the 5.0 flag in soon which can be used to automatically flip this when 5.0 is released